corefoundation.gemspec

```diff
@@ -19,9 +21,9 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.6'
 
-  s.add_runtime_dependency "ffi"
-  s.add_development_dependency "rspec", "~>2.10"
-  s.add_development_dependency "rake"
-  s.add_development_dependency "yard"
-  s.add_development_dependency "redcarpet"
+  s.add_runtime_dependency 'ffi'
+  s.add_development_dependency 'rake'
+  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rubocop'
```

Comment:
We use chefstyle at chef so this needs to be chefstyle and I'd pin to a specific version so you avoid having every new PR break on new linting changed

Reply:
Pinned to 2.0.7 in https://github.com/chef/corefoundation/pull/10/commits/bdeddf5c5b893f9417c03e757a1f1c566ede9c96